src/config.cpp

```diff
@@ -15,11 +15,12 @@
 #include <boost/property_tree/ptree.hpp>
 
 #include "config.h"
+#include "entry_handler.h"
 #include "file_handler.h"
 #include "logging.h"
-#include "main.h"
 #include "nvhttp.h"
 #include "rtsp.h"
+#include "thread_pool.h"
```

Comment:
This was generally added to handle `sleep_for` calls, although I'm not sure it's entirely necessary.

Reply:
You can replace it with `std::this_thread::sleep_for()`